api/src/app.ts

```diff
@@ -105,7 +105,13 @@ export default async function createApp(): Promise<express.Application> {
 
 	app.disable('x-powered-by');
 	app.set('trust proxy', env['IP_TRUST_PROXY']);
-	app.set('query parser', (str: string) => qs.parse(str, { depth: 10 }));
+
+	app.set('query parser', (str: string) =>
+		qs.parse(str, {
+			depth: 10,
```

Comment:
Do we perhaps want to tag this to `MAX_RELATIONAL_DEPTH`? 🤔

Reply:
No, I don't think so as filters could include logical operators which don't count under `MAX_RELATIONAL_DEPTH`. However, it would be nice to have a `QUERY_PARSER_DEPTH_LIMIT` since we can configure array limit. 👍